api/v1beta1/hyperconverged_types.go

```diff
@@ -91,7 +91,7 @@ type HyperConvergedSpec struct {
 
 	// ResourceRequirements describes the resource requirements for the operand workloads.
 	// +kubebuilder:default={"vmiCPUAllocationRatio": 10}
-	// +kubebuilder:validation:XValidation:rule="!has(self.vmiCPUAllocationRatio) || self.vmiCPUAllocationRatio != 1",message="Automatic CPU limits are incompatible with a VMI CPU allocation ratio of 1"
+	// +kubebuilder:validation:XValidation:rule="self.vmiCPUAllocationRatio == null || self.vmiCPUAllocationRatio > 0",message="vmiCPUAllocationRatio must be greater than 0"
```

Comment:
I think the right syntax is `has(x)`, rather than `x == null`

Reply:
Fixed, thanks